sql-db/sql-reactive/src/test/java/io/quarkus/ts/openshift/reactive/Postgresql12DatabaseIT.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.ts.openshift.reactive;
+
+import io.quarkus.test.bootstrap.DefaultService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+//@EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")
+public class Postgresql12DatabaseIT extends AbstractReactiveDatabaseIT {
```

Comment:
You're using Postgres SQL 10, so you don't need the RH registry and this comment can be out, plus rename the class name to Postgresql10DatabaseIT.

Reply:
You're right, fixed